internal/pipeline/component/pipeline_test.go

```diff
@@ -243,6 +413,48 @@ spec:
                 database: ${parameters.database}
   workload: {}
 `,
+			environmentYAML: `
```

Comment:
If these doesn't change between test cases, shall we extract this to top of the test/file?

Reply:
Resolved with [Reuse environment and dataplane YAMLs in test cases](https://github.com/openchoreo/openchoreo/pull/783/commits/486e290ee0f699b91cd35416ace908017295163b)